src/rpc/net.cpp

```diff
@@ -544,16 +544,20 @@ UniValue setban(const JSONRPCRequest& request)
             absolute = true;
 
         if (isSubnet) {
-            g_connman->Ban(subNet, BanReasonManuallyAdded, banTime, absolute);
-            g_connman->DisconnectNode(subNet);
+            g_banman->Ban(subNet, BanReasonManuallyAdded, banTime, absolute);
+            if (g_connman) {
```

Comment:
Is it really worth checking? We already check for g_banman further up, are we really anticipating having a g_banman but not a g_connman?

Reply:
Yea, I think it may be useful in the future to run with -p2p=0 in order to view/cleanup your bandb.